backend/peephole/peephole_utils.mli

```diff
@@ -0,0 +1,24 @@
+[@@@ocaml.warning "+a-29-40-41-42"]
+
+module DLL = Flambda_backend_utils.Doubly_linked_list
+
+val are_equal_regs : Reg.t -> Reg.t -> bool
+
+val go_back_const : int
```

Comment:
The semantics of most values can
be inferred from their names, but I
think `go_back_const` and `rule`
would deserve a docstring.

Reply:
We agreed to only ad a docstring for `rule` as `go_back_const` will be deleted soon.